src/hooks/useBodyScrollLock/index.ts

```diff
@@ -0,0 +1,2 @@
+export {useBodyScrollLock} from './useBodyScrollLock';
+export type {BodyScrollLockProps} from './useBodyScrollLock';
```

Comment:
New interface should be defined and aliased to old one:
```ts
interface UseBodyScrollLockProps {}

type BodyScrollLockProps = UseBodyScrollLockProps
```

And both should be exported

Reply:
fix